src/pages/schedule.tsx

```diff
@@ -1,7 +1,10 @@
+import Image from "next/image";
+import streamSchedulePhoto from "../../public/images/schedule.png";
+
 const Schedule = () => {
   return (
     <div className="flex flex-col items-center">
-      <img src="/images/schedule.png" className="max-w-xl" alt="stream schedule" />
+      <Image src={streamSchedulePhoto} className="max-w-xl" alt="stream schedule" />
```

Comment:
We can't use the next/image Image, because of the way the website is exported. Image optimisation is not supported for next export. 
`https://nextjs.org/docs/api-reference/next/image#built-in-loaders`

Reply:
Okay, noted.